tests/integration/test_strict_config.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -eou pipefail
+
+set +e
+bazelisk clean
+bazelisk build //tests/integration:detekt_with_strict_config > /tmp/bazel.log > /dev/null
+BAZEL_EXIT_CODE=$?
+set -e
+
+test BAZEL_EXIT_CODE != 0
```

Comment:
Just let `set -e` handle it, I get that you kinda making test more explicit by checking with `test` but in fact it's less safe than just running the script with `set -e` as I noted above

Reply:
Hm, but I’m checking that the execution failed and that’s a correct behavior. Successful execution would be a failing condition in this case.

Reply:
oh, my bad, now I see 👍